readme.md

```diff
@@ -7,11 +7,16 @@
 
 ginh is not a histogram
 
-usage: `./ginh.sh [-h] [-n entries] [-f hist_file] [-c chart_char] [-l line_len]`
+usage: `./ginh.sh [-h] [-a] [-n entries] [-f hist_file] [-c chart_char] [-l line_len]`
```

Comment:
Do you think it would be a good idea to add the explanation of these flags to the help/usage output?

Reply:
check lines 16-19